src/main/resources/templates/customerpage.html

```diff
@@ -30,8 +30,13 @@
         </div>
         <div id="cartbox">
             <!-- here we have som links to both the login in site and to the cart. these doesn't need to be links, we can use SPA if we want. -->
-            <div id="login"><a href="registration.html"><img src="./img/loginlogo.png" id="loginlogo"/></a></div>
-            <div id="cart" ><img src="./img/shopping-cart.png" id="cartlogo"/></div>
+            <div id="login"><a href="registration.html"><img src="../static/img/mypagelogo.png" id="mypagelogo"/></a></div>
```

Comment:
Should not be ../static/img/mypagelogo.png. 
change to either 
/img/mypagelogo.png 
or 
./img/mypagelogo.png

Reply:
ok.... I changed a bit